dearpygui/demo.py

```diff
@@ -1524,10 +1550,24 @@ def _scroll_programmatically(sender, app_data, user_data):
                                             "pad_outerX", "no_pad_outerX", "no_pad_innerX", 
                                             "borders_outerV", "borders_innerV", "header_row", before=table_id)
 
+            with dpg.tree_node(label="Angled headers"):
+
+                with dpg.table(header_row=True, resizable=True, delay_search=True,
+                            reorderable=True) as table_id:
+
+                    dpg.add_table_column(label="One", angled_header=True)
+                    dpg.add_table_column(label="Two", angled_header=True)
+                    dpg.add_table_column(label="three", angled_header=True)
+
+                    for i in range(5):
+                        with dpg.table_row():
+                            for j in range(3):
+                                dpg.add_text(f"Hello {i}, {j}")
+
             with dpg.tree_node(label="Reorderable, hideable, with headers"):
 
                 with dpg.table(header_row=True, resizable=True, delay_search=True,
-                            hideable=True, reorderable=True) as table_id:
```

Comment:
As I mentioned in [another comment](https://github.com/hoffstadt/DearPyGui/pull/2275#discussion_r1651478981), `hideable=True` got removed where it probably shouldn't. This line is particularly interesting: the table resides in the section named "Reorderable, hideable, with headers", but the columns are now _not hideable_ by default :joy:.

Also see other occurrences (just search for "hideable=True" through the diff).

Reply:
I see that _some_ `hideable=True` got restored, but not all. For example, `hideable=True` in "Column Options" allowed to test the `no_hide` column option. Now, without `hideable=True`, `no_hide` does nothing. Can you please look at the diff and revert _all_ changes to the hideable attribute? I don't think we need to remove it where it used to be.